### PR TITLE
Fully Developed Application

### PR DESCRIPTION
**Pull request by El-Potato-Slayer**
- Contains a home, menu, and contact page
- Each page is loaded without to refresh
- Contains 3 logic modules: parallax.js, navbar.js, and tab.js
- parallax.js and navbar.js are responsible for visual animations
- tab.js is responsible for displaying the content of which page is selected
- Each page and logic have been separated into their respective modules
- All modules are exported in index.js
- Fully responsive

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (wathigo) [inline .vscode/settings.json:3]: **REQUIRED**
* Files not related to the project should not be committed and added to .gitignore (e.g. .DS_Store) ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png))

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "js-restaurant",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (wathigo) [inline package.json]: **REQUIRED**
Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png))

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o:  
Great job on this project.
Let's make a couple of changes as shown in the comments :point_down: 

**After making the changes, make another request for review.**

**Happy Coding :computer_mouse: :technologist: :computer: 
Simon Wathigo**

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**